getnacelle/nacelle-js
# chore(Commerce Queries Plugin): exports all of the generated Storefront GQL Types

## Why are these changes introduced?

Addresses [ENG-8955](https://nacelle.atlassian.net/browse/ENG-8955) 



## What is this pull request doing?

- Re-exports all of the types in `types/storefront.ts` to make them available to users. 



## How to Test

1. Checkout this branch `mdarrik/ENG-8955-export-commerce-queries-types`
2. Navigate to `packages/storefront-sdk-plugins/commerce-queries` and run `npm run build` (should build successfully)
3. Use `npm link @nacelle/commerce-queries-plugin` in an external typescript project. Non-toplevel GQL types like `Variant` should be importable from `@nacelle/commerce-queries-plugin`



## Checklist



[ENG-8955]: https://nacelle.atlassian.net/browse/ENG-8955?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
